anndata/_core/views.py

```diff
@@ -100,6 +108,13 @@ class DictView(_ViewMixin, dict):
 class DataFrameView(_ViewMixin, pd.DataFrame):
     _metadata = ["_view_args"]
 
+    @wraps(pd.DataFrame.drop)
+    def drop(self, *args, inplace: bool = False, **kw):
+        if not inplace:
+            return super().drop(*args, **kw)
```

Comment:
The test for this case still breaks … no idea how to elegantly and correctly do this.